AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs

```diff
@@ -248,9 +257,13 @@ public void NavigateBack()
 
         public string TextOf(By element)
         {
-            var text = string.Empty;
-            RetryOnStaleElement(() => text = PerformGetText(element));
-            return text.IsNullOrEmpty() ? text : text.Trim();
+            WebDriverWait wait = new WebDriverWait(Driver, timeout: TimeSpan.FromSeconds(30))
+            {
+                PollingInterval = TimeSpan.FromSeconds(1),
```

Comment:
Can we make this one method that we call with different params as we have some repeated code in here

Reply:
I'm going to convert this to a helper class as it's in two classes and may be used in more later.